.github/workflows/codeql.yml

```diff
@@ -16,7 +16,7 @@ on:
     branches: [ "master" ]
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [ "master" ]
+    branches: [ "master", "fix-codeql" ]
```

Comment:
what is this?

Reply:
cant find when it was added, must be me polluted my master branch.